products/lineageos.md

```diff
@@ -13,28 +13,56 @@ discontinuedColumn: false
 
 releases:
 -   releaseCycle: "20"
+    releaseLabel: "20"
```

Comment:
```suggestion
```

The `releaseLabel` is not needed as it's the same of the `releaseCycle`.

Reply:
right